packages/client/src/Queries.ts

```diff
@@ -145,6 +145,7 @@ function isQueryRange<T extends QueryRangeOptions>(options: any): options is T {
 export type QueryOptions = {
 	abortSignal?: AbortSignal;
 	verifyNetworkResponses?: VerificationOptions | boolean;
+	raw?: boolean;
```

Comment:
What is the significance of passing `raw` and using `PushBuffer`?

Reply:
Passing raw here, means the LogStoreClient won't decrypt the messages queried from the LogStore plugin of a broker.
Instead the LogSroreClient will forward the messages through PushBuffer to the querying StreamrClient letting it to decrypt the messages by its own. Doing so the LogStoreClient as a core thing of the StorageProxy cannot access encrypted messages, it just forwards whatever returned by the broker.